src/library/trackmodel.h

```diff
@@ -144,7 +147,26 @@ class TrackModel {
     virtual void select() {
     }
 
+    virtual int getPosition(){
+      return m_position;
+    }
+
+    virtual void setPosition(int newPosition){
+      // in bug 1450391 we ve got situation
+      // when during changing data model
+      // scrollPositionChanged event is emitted
+      // and scrollBar moves to 15 or 16 from 30-40
+      // check of new value can help to prevent such behaviour
+      // better to find why event is emitted
+      if (newPosition <= 100 && newPosition >= 0 && abs(m_position-newPosition) < positionChangeLimit){
```

Comment:
I have just tried to make this if false by manual test, but I failed. 
What is the receipt for it?